### PR TITLE
Fixed filament sensor on printer-sunlu-t3-2022.cfg

### PR DESCRIPTION
**Pull request by dahliamoth**
Discovered that the filament sensor included with the printer is not a switch, it is a motion sensor.

The filament sensor on the printer used for initial testing had been removed prior to flashing klipper to it. Only a basic test to see it if triggered or not was preformed. It was discovered by someone else doing some testing on another printer that it would frequently pause. This was a simple oversight, assuming it was a runout switch.

Removed section of config related to filament runout switch and replaced with basic config for filament motion so they printer will simply pause when no more motion is detected. Also notated to add CLEAR_PAUSE to start gcode if using this sensor as it will trigger a paused state when loading or unloading filament. If printer is in a paused state at the start of the print, the attempt to pause will cause the printer to rapidly pause and unpause itself.

signed-off-by: Zachary Welvaert zwelvaert@gmail.com

### CODE REVIEW

File: config/printer-sunlu-t3-2022.cfg [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC8
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PA0
+control: pid
+pid_Kp: 19.479 #calibrate your own PID
+pid_Ki: 1.073
+pid_Kd: 88.385
+min_extrude_temp: 175
```
Comment (JamesH1978) [inline config/printer-sunlu-t3-2022.cfg:143]: Please do not override safety settings, min_extruder_temp and max_extrude_only_distance should be removed.

**Review by JamesH1978 — CHANGES_REQUESTED** [review]
Thank you for submitting your printer config.

I have made one comment so far, change this and I'll do another review and see if it's good to approve

thanks James

File: config/printer-sunlu-t3-2022.cfg (hunk 1 above)
Comment (dahliamoth) [reply config/printer-sunlu-t3-2022.cfg]: Sorry about that. I had those in my own config for use with filament load/unload macros and forgot to remove them when i removed those macros. Removed now.

**Comment by JamesH1978** [comment]
I am not sure why, but your PR has not started the check process

**Comment by dahliamoth** [comment]
> I am not sure why, but your PR has not started the check process

I maybe bc to fix this issue, i edited my old fork that was already merged. I probably needed to delete this fork and suggest the edit on a new fork. I can do that if needed

**Comment by JamesH1978** [comment]
I would assume that is what has happened. I would delete this and use a new fork.

Thanks
James

**Comment by dahliamoth** [comment]
> I would assume that is what has happened. I would delete this and use a new fork.

I closed this PR and opened a new one on a updated fork. It is now autostarting the build test, but apparently at the moment that I submitted it there is a scheduled brownout of Ubuntu 18.04, so the test canceled itself. Hopefully when the brownout if over it will rerun its test

**Pull request closed without merging**